airflow/Dockerfile

```diff
@@ -1,15 +1,46 @@
-#
-# We use the official Airflow image here instead of building our own from scratch
-# to avoid having to deal with python dependencies and Nexus integration. We might
-# change this in the future.
-#
-# 2022.01.20: Some components (e.g. airflow triggerer) require python 3.7+
-#
+FROM registry.access.redhat.com/ubi8/ubi-minimal:8.5 as airflow-build-image
+
 ARG PRODUCT
 ARG PYTHON
-ARG RELEASE="1"
 
-FROM apache/airflow:$PRODUCT-python$PYTHON
+COPY airflow/constraints-${PRODUCT}-python${PYTHON}.txt /tmp/constraints.txt
+
+# the mysql provider is currently excluded.
+# Requires implementation of https://github.com/apache/airflow/blob/2.2.5/scripts/docker/install_mysql.sh
+ENV AIRFLOW_EXTRAS=async,amazon,celery,cncf.kubernetes,docker,dask,elasticsearch,ftp,grpc,hashicorp,http,ldap,google,google_auth,microsoft.azure,odbc,pandas,postgres,redis,sendgrid,sftp,slack,ssh,statsd,virtualenv
+
+RUN microdnf update \
+        && microdnf install -y \
+            gcc \
+            gcc-c++ \
+            python${PYTHON}-devel \
+            python${PYTHON}-pip \
+            python${PYTHON}-wheel \
+            openssl-devel \
+            cyrus-sasl-devel \
+            openldap-devel \
+            unixODBC-devel \
```

Comment:
I think you have to install non devel versions in the final stage for any code that uses them to work.

Reply:
Done. Will rebuild against supported versions and re-run the kuttl tests.